code/modules/reagents/reagents/medicine.dm

```diff
@@ -750,9 +776,9 @@ We don't use this but we might find use for it. Porting it since it was updated
 	taste_description = "bitterness"
 	reagent_state = LIQUID
 	color = "#561EC3"
-	overdose = 10
+	overdose = 20 // Higher overdose vs Vanilla
 	scannable = TRUE
-	nerve_system_accumulations = 30
+	nerve_system_accumulations = 50 // But also higher NSA
```

Comment:
Re this, I'd lower NSA to 40 because currently with 50, 2 doses will cause NSA overcreep.